README.md

```diff
@@ -7,23 +7,70 @@ Copacetic Action is supported with Copa version 0.3.0 and later.
 
 ## Inputs
 
-| Name               | Type   | Required | Default   | Description                                            |
-| ------------------ | ------ | -------- | --------- | ------------------------------------------------------ |
-| `image`            | String | True     |           | Image reference to patch                               |
-| `image-report`     | String | True     |           | Trivy JSON vulnerability report of the image to patch  |
-| `patched-tag`      | String | True     |           | Patched image tag                                      |
-| `timeout`          | String | False    | `5m`      | Timeout for `copa patch`                               |
-| `buildkit-version` | String | False    | `latest`  | Buildkit version                                       |
-| `copa-version`     | String | False    | `latest`  | Copa version                                           |
-| `output`           | String | False    |           | Output filename (available with copa v0.5.0 and later) |
-| `format`           | String | False    | `openvex` | Output format (available with copa v0.5.0 and later)   |
+| Name               | Type   | Required | Default   | Description                                                                |
+| ------------------ | ------ | -------- | --------- | -------------------------------------------------------------------------- |
+| `image`            | String | True     |           | Image reference to patch                                                   |
+| `image-report`     | String | True     |           | Trivy JSON vulnerability report of the image to patch                      |
+| `patched-tag`      | String | True     |           | Patched image tag                                                          |
+| `timeout`          | String | False    | `5m`      | Timeout for `copa patch`                                                   |
+| `buildkit-version` | String | False    | `latest`  | Buildkit version                                                           |
+| `copa-version`     | String | False    | `latest`  | Copa version                                                               |
+| `output`           | String | False    |           | Output filename (available with copa-action v0.6.1 and later)              |
+| `format`           | String | False    | `openvex` | Output format (available with copa-action v0.6.1 and later)                |
+| `custom-socket`    | String | False    |           | Custom Docker socket address (available with copa-action v0.6.1 and later) |
+
+> [!NOTE] 
+> Copacetic released version's features do not align with Copa Action versions.
```

Comment:
This is a bit confusing. Are we saying copa may have features that action might not?

Reply:
I meant to say that if a feature was released in Copa version v0.6.0 (like patching local images), it wont necessarily be supported in Copa-Action version v0.6.0. In this case it will supported starting with Copa-Action v0.6.1. I changed the wording a bit to try to make it more clear.